driver/razermouse_driver.c

```diff
@@ -4666,6 +4686,139 @@ static void razer_mouse_init(struct razer_mouse_device *dev, struct usb_interfac
     dev->tilt_repeat = 33;
 }
 
+int razer_has_battery(unsigned short usb_pid)
```

Comment:
Can we get this sort of dynamically? Anything with is_charging or something along the lines should have that, no?

Or at the very least we need a simple CI check that makes sure we don't forget to add devices here in this function

Reply:
I just manually looked at razer_mouse_probe and got all the devices that created battery attributes to create that function but ideally the code should be refactored or there should be a data structure to avoid the duplicated code.